.github/workflows/generate-code-owners.yml

```diff
@@ -0,0 +1,32 @@
+name: Update CODEOWNERs
+
+on:
+  schedule:
+    # https://crontab.guru/#5_8_*_*_1
+    - cron: "5 8 * * 1"
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    if: github.repository == 'raycast/extensions'
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
```

Comment:
I suggest to  use here our custom [github-action](https://github.com/raycast/github-actions/blob/master/setup-node/action.yml) that was will allow us to keep all our node version being the same all across our scripts. Unless there is a reason to keep this one separated.